sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -23,7 +23,7 @@ Use the client library for App Configuration to create and manage application co
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-data-appconfiguration</artifactId>
-  <version>1.1.5</version>
+  <version>1.1.6</version>
 </dependency>
```

Comment:
It would be good to get the AAD authentication for AppConfig in sync with other packages content.

Reply:
do you mean to have prefer to use AAD instead of connection string for testing?

Reply:
No, just suggesting to make the AAD auth section of AppConfig similar to other packages AAD section.

Reply:
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration#use-aad-token

is this one?

Reply:
I would keep the  AAD description in app config more similar to key vault now. https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/keyvault/azure-security-keyvault-keys#createget-credentials